internal/driver/commands.go

```diff
@@ -131,6 +131,8 @@ var pprofVariables = variables{
 	// Filename for file-based output formats, stdout by default.
 	"output": &variable{stringKind, "", "", helpText("Output filename for file-based outputs")},
 
+	"comment": &variable{stringKind, "", "", helpText("Append comment to profile before processing to output format")},
```

Comment:
The only output format we mean here is I think profile proto... At least that's the only thing we test. So maybe something like `String to record as a comment for profile proto output`?